### PR TITLE
module-3-day-4 Tasks updated

### PR DESCRIPTION
**Pull request by farooqgollapallishaik**
Module-3 day-4 tasks updated.
1&2.Normal Table, using col span, & row span.
3&4.Nested Table, using col span, & row span.
5.Two nested  tables inside a table.

### CODE REVIEW

File: Module-3/Day4/Task/Farooq/Task2.html
Comment (Saddamkhanlavani) [inline Module-3/Day4/Task/Farooq/Task2.html:1]: Kindly correct this task.

image

Comment (Saddamkhanlavani) [inline Module-3/Day4/Task/Farooq/Task2.html:1]: This is the actual output.

Task-6

File: Module-3/Day4/Task/Farooq/Task3.html
Comment (Saddamkhanlavani) [inline Module-3/Day4/Task/Farooq/Task3.html:1]: This task has been repeated two times in this PR. This task code gives wrong one and task-5 code has the right output for this task so remove this and the missing task was the below one. kindly complete it.

Comment (Saddamkhanlavani) [inline Module-3/Day4/Task/Farooq/Task3.html:1]: Task-3
 
complete this one.

**Comment by abdulhaseebshaik** [comment]
instead of applying border manually to every cell use table attribute border.

**Review by IshaaqhShaik — APPROVED** [review]

**Pull request merged**